components/top-nav/src/atoms/top-nav-inner/index.js

```diff
@@ -9,10 +9,10 @@ const TopNavInner = styled('div')({
   flexDirection: 'column',
   width: `calc(100% - ${SPACING.SCALE_3})`,
   maxWidth: '960px',
-  padding: `${SPACING.SCALE_2} ${SPACING.SCALE_3}`,
+  padding: `8px ${SPACING.SCALE_3}`,
```

Comment:
not sure about the magic number here of `8px` - it feels like this should be a variable, given the other use of variables for spacing